JsonPreprocessor/CJsonPreprocessor.py

```diff
@@ -312,7 +312,7 @@ def replacer(match):
             else:
                 return s
       
-        file=open(jsonFile,mode='r')
+        file=open(jsonFile, mode='r', encoding='utf-8')
```

Comment:
Hi Son,
I wonder what happens if BOM is missing.
utf-8 is optional, but I think the "open" function works already like this. Means if the file is not UTF-8 or BOM is missing it will read properly.
Please test.
Thank you,
 Thomas

Reply:
Hello Thomas,

I already run the acceptance test which I created in before, and all 35 test cases were passed. It means this change works fine with not utf-8 files.

I don't know what is the BOM you mentioned in your comment? Could you please give me some hints?

Thank you,
Son